manifests/c/Corsair/iCUE/4/4.15.153/Corsair.iCUE.4.installer.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Corsair.iCUE.4
+PackageVersion: 4.15.153
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
```

Looks like they changed the switches; I tried with /passive for the silent with progress and kept getting an error that 
> Another program is being installed. Please wait until that installation is complete, and then try installing this software again.
```suggestion
  Silent: /quiet
  SilentWithProgress: /quiet
```

---

Yuup, winget-cli uses /quiet and /passive as default switches for msi's but /passive have always thrown that error and IIRC /quiet didnt work as expected either, however i just tested on my VM and /quiet requires Administrator or it will throw exit code 1603

Ez fix for Manifest Schema 1.1.0 tho